tools/tpm2_nvwrite.c

```diff
@@ -147,8 +150,11 @@ static bool nv_write(TSS2_SYS_CONTEXT *sapi_context) {
 
 static bool on_option(char key, char *value) {
     bool result;
-
+    ctx.hmac_auth = true;//DEBUG
```

Comment:
This seems to be a left over, right?

Reply:
@martinezjavier was hitting some weird issues with command option not being parsed/ recognized. Yes it is a left over.

Reply:
@martinezjavier, this is no longer an option. The choice to use HMAC-auth instead of plaintext password is now specified in the auth utility. So you can specify like -P hmac:<Text>